<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMadrasah</class>
 <widget class="QDialog" name="FormMadrasah">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Madrasah Ibtidaiyah</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="inputNama">
     <property name="placeholderText">
      <string>Nama Madrasah</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputAlamat">
     <property name="placeholderText">
      <string>Alamat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboKecamatan">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string extracomment="Kecamatan"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputGuru">
     <property name="placeholderText">
      <string>Jumlah Guru</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputSiswa">
     <property name="placeholderText">
      <string>Jumlah Siswa</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputTahun">
     <property name="placeholderText">
      <string>Tahun Berdiri</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBatal">
       <property name="text">
        <string>Batal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
